packages/js/src/browser.ts

```diff
@@ -1,169 +1,200 @@
-import { Types, Util, Client } from '@honeybadger-io/core'
-import { encodeCookie, decodeCookie, preferCatch } from './browser/util'
-import { onError, ignoreNextOnError } from './browser/integrations/onerror'
-import onUnhandledRejection from './browser/integrations/onunhandledrejection'
-import breadcrumbs from './browser/integrations/breadcrumbs'
-import timers from './browser/integrations/timers'
-import eventListeners from './browser/integrations/event_listeners'
+import { Types, Util, Client } from '@honeybadger-io/core';
```

Comment:
Hmm, we have to decide on a convention about semi-colons (`;`) at the end of the statements.
So far, we've been avoiding them, but you may find some in some places.
Let's decide on it and we can create a separate PR with an eslint rule to enforce it.

Note: I didn't take this decision, I just went along with the convention :)

cc @joshuap @BethanyBerkowitz @stympy

Reply:
@subzero10 I piped these through prettier, but repiped them back through ts-standard.

IIRC most of our repos use either "standard" or "js-standard"

Reply:
The monorepo has a global eslint configuration. I created the rules based on different styles I picked up from our repos while implementing it. It's quite possible I missed or mixed some. Nothing's set it stone though, we can always improve.

Reply:
@subzero10 the problem was I've recently remove "expandtab" from my Vim config and for some reason ESLint doesnt fix the "mixed spaces + tabs" issues and going in and fixing it manually is a pain. I just reverted my Vim config to "expandtab" for now